instrument/Facilities.xml

```diff
@@ -23,140 +24,179 @@
+    <livedata>
+      <connection name="histo" address="NDXIMAT:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="INES" shortname="INS" beamline="N8">
     <technique>Neutron Diffraction</technique>
-    <livedata address="NDXINES:6789" />
+    <livedata>
+      <connection name="histo" address="NDXINES:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="INTER">
     <technique>Reflectometry</technique>
     <zeropadding size="8"/>
-    <livedata address="NDXINTER:6789" />
+    <livedata>
+      <connection name="histo" address="NDXINTER:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="IRIS" shortname="IRS" beamline="N6">
     <zeropadding size="8" startRunNumber="54938" prefix="IRIS"/>
     <technique>Neutron Spectroscopy</technique>
     <technique>TOF Indirect Geometry Spectroscopy</technique>
-    <livedata address="NDXIRIS:6789" />
+    <livedata>
+      <connection name="histo" address="NDXIRIS:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="LARMOR">
     <technique>Small Angle Scattering</technique>
     <zeropadding size="8"/>
-    <livedata address="NDXLARMOR:6789" />
+    <livedata>
+      <connection name="histo" address="NDXLARMOR:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="LOQ">
     <technique>Small Angle Scattering</technique>
-    <livedata address="NDXLOQ:6789" />
+    <livedata>
+      <connection name="histo" address="NDXLOQ:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="MAPS" shortname="MAP">
     <technique>Neutron Spectroscopy</technique>
     <technique>TOF Direct Geometry Spectroscopy</technique>
-    <livedata address="NDXMAPS:6789" />
+    <livedata>
+      <connection name="histo" address="NDXMAPS:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="MARI" shortname="MAR">
     <technique>Neutron Spectroscopy</technique>
     <technique>TOF Direct Geometry Spectroscopy</technique>
-    <livedata address="NDXMARI:6789" />
+    <livedata>
+      <connection name="histo" address="NDXMARI:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="MERLIN" shortname="MER">
     <technique>Neutron Spectroscopy</technique>
     <technique>TOF Direct Geometry Spectroscopy</technique>
-    <livedata address="NDXMERLIN:6789" />
+    <livedata>
+      <connection name="histo" address="NDXMERLIN:6789" listener="ISISHistoDataListener" />
+    </livedata>
   </instrument>
 
   <instrument name="MERLIN_EVENT" shortname="MER">
     <technique>Neutron Spectroscopy</technique>
     <technique>TOF Direct Geometry Spectroscopy</technique>
-    <livedata address="NDXMERLIN:10000" listener="ISISLiveEventDataListener" />
+    <livedata>
```

Comment:
Please create a new Issue for this if you haven't already and assign it to yourself, and of course link the issues together.